src/main/groovy/com/avioconsulting/mule/linter/model/VersionCompare.groovy

```diff
@@ -0,0 +1,28 @@
+package com.avioconsulting.mule.linter.model
+
+class VersionCompare {
```

Comment:
This would probably be "Version" as a model class, which represents a specific version, and provides methods to compare if it is equal, greater than, etc.. compared to another version

Reply:
Ya I can do that.